### ISSUE TITLE
Parse NHANES lab panel into individual lab tests

### ISSUE THREAD

**Issue by rehoyt** [LibreHealthIO/LibreEHR#655]

**Comment by teryhill** [comment]
Is something that @yehster could look at doing.

**Comment by yehster** [comment]
Yes, this one is on my plate. It's really an issue with the import process and not the EHR itself.

**Comment by rehoyt** [comment]
The other issue I think I posted in the forum is can we set a default search date (the From field) for all reports and set it before 2016-12-31. If students forget to do this they may not get any results back
